### PR TITLE
Update section data for N4993

### PR DESCRIPTION
**Pull request by CaseyCarter**
N4993 rearranged many of the clauses and subclauses, so there's quite a bit of activity in `section_data`. Note that the format of the `aux` files has also seemingly changed, I had to update the filters quite a bit.

### CODE REVIEW

**Comment by Dani-Hub** [comment]
Ah great, I was already considering to ask Jonathan about the update!

File: Makefile [hunk 1]
```diff
@@ -72,13 +72,13 @@ endef
 WG21 := $(HOME)/src/cplusplus
 DRAFT := $(WG21)/draft
 NET := $(WG21)/networking-ts
-filter-annex-f := sed 's/\\newlabel{\([^}]*\)}.*TitleReference {\([^}]*\)}.*/\1 \2/' | sed 's/\\newlabel{\([^}]*\)}{{\(Clause\|Annex\) \([^}]*\)}.*/\1 \3/' | grep -v "aux:tab:" | grep -v "aux:fig:" | sed 's/\(.*\).aux://' | grep -v '^\\' | sort
+filter-annex-f := grep -v '\\newlabel{\(fig\|tab\):' | sed 's/^.*\.aux://' | sed 's/^\\newlabel{\([^}]*\)}{{\([^}]*\)}.*/\1 \2/' | grep -v '^\\' | sed 's/\(Clause\|Annex\) //' | sort
```
Comment (timsong-cpp) [inline Makefile:75]: Are you sure this is the right filter? It's producing lines like `adjacent.difference \M@TitleReference {26.10.12` for me.

Comment (timsong-cpp) [reply Makefile:75]: The original formula still works for me.

Comment (CaseyCarter) [reply Makefile:75]: Different `aux` file contents due to different versions of LaTeX, or some LaTeX package, maybe? I have no occurrences of `TitleReference` or `^(fig|tab):` in `*.aux` (attached as [n4993.zip](https://github.com/user-attachments/files/17592021/n4993.zip)). The original filters produce a `meta-data/annex-f` from only `Clause|Annex` hits:
```
algorithms 26
basic 6
class 11
concepts 18
containers 23
cpp 15
dcl.dcl 9
depr D
diagnostics 19
diff C
except 14
exec 33
expr 7
gram A
implimits B
input.output 31
intro 4
intro.defs 3
intro.refs 2
intro.scope 1
iterators 24
lex 5
library 16
mem 20
meta 21
module 10
numerics 29
over 12
ranges 25
stmt.stmt 8
strings 27
support 17
temp 13
text 28
thread 32
time 30
uaxid E
utilities 22
```
FWIW, I'm generating in Ubuntu-24.04.1, IIRC with whatever `apt-get install texlive-meow` gave me.

Comment (CaseyCarter) [reply Makefile:75]: Yes, this formulation also works for me at `main`:
```diff
diff --git a/meta-data/section.data b/meta-data/section.data
index e242b1bf13..607c03e978 100644
--- a/meta-data/section.data
+++ b/meta-data/section.data
@@ -79,15 +79,15 @@
     5.3 [lex.char]
         5.3.1 [lex.charset]
         5.3.2 [lex.universal.char]
-    5.4 [lex.pptoken]
-    5.5 [lex.digraph]
-    5.6 [lex.token]
-    5.7 [lex.comment]
-    5.8 [lex.header]
-    5.9 [lex.ppnumber]
-    5.10 [lex.name]
-    5.11 [lex.key]
-    5.12 [lex.operators]
+    5.4 [lex.comment]
+    5.5 [lex.pptoken]
+    5.6 [lex.header]
+    5.7 [lex.ppnumber]
+    5.8 [lex.operators]
+    5.9 [lex.digraph]
+    5.10 [lex.token]
+    5.11 [lex.name]
+    5.12 [lex.key]
     5.13 [lex.literal]
         5.13.1 [lex.literal.kinds]
         5.13.2 [lex.icon]
@@ -139,7 +139,6 @@
                 6.7.5.5.1 [basic.stc.dynamic.general]
                 6.7.5.5.2 [basic.stc.dynamic.allocation]
                 6.7.5.5.3 [basic.stc.dynamic.deallocation]
-            6.7.5.6 [basic.stc.inherit]
         6.7.6 [basic.align]
         6.7.7 [class.temporary]
     6.8 [basic.types]
@@ -3343,27 +3342,27 @@ C [diff]
             C.8.5.3 [diff.malloc]
 D [depr]
     D.1 [depr.general]
-    D.2 [depr.capture.this]
-    D.3 [depr.array.comp]
+    D.2 [depr.local]
+    D.3 [depr.capture.this]
     D.4 [depr.volatile.type]
-    D.5 [depr.static.constexpr]
-    D.6 [depr.local]
-    D.7 [depr.impldec]
+    D.5 [depr.array.comp]
+    D.6 [depr.impldec]
+    D.7 [depr.static.constexpr]
     D.8 [depr.lit]
     D.9 [depr.template.template]
     D.10 [depr.numeric.limits.has.denorm]
     D.11 [depr.c.macros]
-    D.12 [depr.relops]
-    D.13 [depr.cerrno]
-    D.14 [depr.meta.types]
+    D.12 [depr.cerrno]
+    D.13 [depr.meta.types]
+    D.14 [depr.relops]
     D.15 [depr.tuple]
     D.16 [depr.variant]
     D.17 [depr.iterator]
     D.18 [depr.move.iter.elem]
-    D.19 [depr.format]
-        D.19.1 [depr.format.syn]
-        D.19.2 [depr.format.arg]
-    D.20 [depr.locale.category]
+    D.19 [depr.locale.category]
+    D.20 [depr.format]
+        D.20.1 [depr.format.syn]
+        D.20.2 [depr.format.arg]
     D.21 [depr.fs.path.factory]
     D.22 [depr.atomics]
         D.22.1 [depr.atomics.general]
```

Comment (timsong-cpp) [reply Makefile:75]: Different version of `memoir` I think.

Comment (CaseyCarter) [reply Makefile:75]: My memoir is:
```
(/usr/share/texlive/texmf-dist/tex/latex/memoir/memoir.cls
Document Class: memoir 2024/01/26 v3.8.2 configurable book, report, article doc
ument class
```
which is current according to https://ctan.org/pkg/memoir?lang=en.

Comment (timsong-cpp) [reply Makefile:75]: Yep, older memoir will produce the `\M@TitleReference` thing which is removed in 3.8.

Comment (CaseyCarter) [reply Makefile:75]: Resolved by #436.

**Pull request merged**